docs/source/contrib_guide_docs.rst

```diff
@@ -1,3 +1,108 @@
+========================================
+Submitting a Bug or Enhancement Request
```

Comment:
I would split bugs and enhancements into two different sections.

Bugs - It would be helpful to guide folks to resources to search project issues and Jupyter mailing list as well as general search before filing an issue.

Enhancements - Encourage informal discussion of ideas on mailing list and submission of an issue as an enhancement on GitHub.